app/nl/ooot/wms/graphql/GraphQLController.scala

```diff
@@ -0,0 +1,91 @@
+package nl.ooot.wms.graphql
+
+import akka.actor.ActorSystem
+import controllers.AssetsFinder
+import javax.inject.Inject
+import play.api.Configuration
+import play.api.libs.json._
+import play.api.mvc._
+import sangria.execution._
+import sangria.marshalling.playJson._
+import sangria.parser.{QueryParser, SyntaxError}
+import sangria.renderer.SchemaRenderer
+import sangria.slowlog.SlowLog
+
+import scala.concurrent.Future
+import scala.util.{Failure, Success}
+
+class GraphQLController @Inject()(system: ActorSystem, config: Configuration)(implicit assetsFinder: AssetsFinder)
+  extends InjectedController {
+
+  import system.dispatcher
+
+  val defaultGraphQLUrl = config.getOptional[String]("defaultGraphQLUrl")
+    .getOrElse(s"http://localhost:${config.getOptional[Int]("http.port").getOrElse(9000)}/graphql")
```

Comment:
CONFIG

Reply:
As in, protocol and host should be configurable. Port looks as if it already is, thats good.